README.md

```diff
@@ -58,6 +58,7 @@ journalctl -fu docker.service # Journalctl/Systemd
  and log location. Example:
 ```
  {"MaxLogAgeDays": 28,
+ "MaxLogFiles": 3,
```

Comment:
Why not make this 10?

Reply:
This is just an example value for the reference and customer/user has to supply appropriate value as per need.

Reply:
Even though these values are configurable we need to set a sensible default value.
I suggested the settings that most of the services in vSphere use as it seems to work well for vSphere customers.

Reply:
sure!